design/ceph/object/store.md

```diff
@@ -351,3 +351,212 @@ spec:
+
+Ceph administrator creates set of `index_pool`, `data_pool`, and `data_extra_pool` per each target placement. Select arbitrary names for target placements. For example `hot` and `cold`:
+```yaml
+# Bucket Data pool for hot storage
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: hot.rgw.buckets.data
+  namespace: rook-ceph 
+spec:
+  ...
+---
+# Bucket index pool for hot storage
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: hot.rgw.buckets.index
+  namespace: rook-ceph
+spec:
+  ...
+---
+# Bucket data extra pool for hot storage
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: hot.rgw.buckets.non-ec
+  namespace: rook-ceph
+spec:
+  ...
+---
+# Bucket Data pool for cold storage
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: cold.rgw.buckets.data
+  namespace: rook-ceph 
+spec:
+  ...
+---
+# Bucket index pool for cold storage
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: cold.rgw.buckets.index
+  namespace: rook-ceph
+spec:
+  ...
+---
+# Bucket data extra pool for cold storage
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: cold.rgw.buckets.non-ec
+  namespace: rook-ceph 
+spec:
+  ...
+```
+If needed, create additional pool to use it in custom StorageClass, for example `GLACIER`:
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: glacier.rgw.buckets.data
+  namespace: rook-ceph 
+spec:
+  ...
+```
+Then update `CephObjectStore` CRD with target placements and StorageClasses definitions:
+```yaml
+spec:
+  targetPlacements:                         # [optional] list of target placements
+  # target placement named hot:
+  - name: hot                               # [required] should be unique
+    dataPool: hot.rgw.buckets.data          # [required] default data pool, will be used with Ceph default STANDARD StorageClass, can be overridden with custom StorageClasses
+    indexPool: hot.rgw.buckets.index        # [required] 
+    dataExtraPool: hot.rgw.buckets.non-ec   # [required] 
+    default: false                          # true|false only one target must be set as default
+  # target placement named cold
+  - name: cold                              # [required] should be unique
+    dataPool: cold.rgw.buckets.data         # [required] default data pool, will be used with Ceph default STANDARD StorageClass, can be overridden with custom StorageClasses
+    indexPool: cold.rgw.buckets.index       # [required] 
+    dataExtraPool: cold.rgw.buckets.non-ec  # [required] 
+    default: true                           # true|false only one target must be set as default
+    storageClasses:                         # [optional] list of custom StorageClasses
+    - name: GLACIER                         # [required]
+      dataPool: glacier.rgw.buckets.data    # [required] should be different from target dataPool
```

Comment:
We may need to also plan to add `storageClasses` to the default pool selection as well, especially if we make the top-level selection the default as I believe may make the most sense.